tests/e2e/common.go

```diff
@@ -107,3 +116,33 @@ func printPodLogs(client clientgo.Interface, namespace, podName, containerName s
 	logBytes, _ := io.ReadAll(stream)
 	fmt.Printf("Printing Log for namespace=%q, pod=%q, container=%q:\n%s\n", namespace, podName, containerName, string(logBytes))
 }*/
+
+func getPodLogs(client clientgo.Interface, namespace, labelSelector, containerName, fileName string) {
+
+	ctx := context.Background()
+	podLogOptions := &corev1.PodLogOptions{Container: containerName}
+
+	podList, err := client.CoreV1().Pods(namespace).List(ctx, metav1.ListOptions{LabelSelector: labelSelector})
+	if err != nil {
+		// TODO: improve error reporting
```

Comment:
Do you want to just do this now instead of needing to have a TODO? Maybe you could just return an error? or did you have something else in mind?

Reply:
Ack, I just left this as a reminder to add in the PR while I was just testing things out quickly. I think we could just return the error as well. We just want to be sure we get to stopping the test env or aborting the suite.